user-auth/README.md

```diff
@@ -0,0 +1,70 @@
+# Rust SGX - Template project
+==================================
+
+### This is a template project to start developing with the Rust SGX SDK (https://github.com/apache/incubator-teaclave-sgx-sdk/) easily.
+
+You will find in its template:
+- Makefiles to build your project easily, and link the ```SGX EDL C``` generated files to your Rust SGX projects
+- The file ```buildenv.mk``` that contains compilation rules when building enclave. No need to specify anymore where this file is located.
+- The file ```build.rs``` already configured to build the app/host part properly.
+- The file rust-toolchain, so we can force the use of one specific toolchain (```nightly-2020-10-25``` in this case)
+- ```Cargo/Xargo.toml``` files to set up your project easily. All the dependencies you might need has been added.
+
+You can find those files in this template: 
+
+```
+|-- app/
+|   |-- src/
+|       |-- main.rs
+|   |-- Cargo.toml
+|   |-- Makefile
+|   |-- build.rs
+|   +-- rust-toolchain
+|-- enclave/
+|   |-- src/
+|       |-- lib.rs
+|   |-- Cargo.toml
+|   |-- Enclave.config.xml
+|   |-- Enclave.edl
+|   |-- Enclave.lds
+|   |-- Makefile
+|   |-- Xargo.toml
+|   +-- rust-toolchain
```

Comment:
same here, to avoid duplicating this info

Reply:
This is a draft only.  The final version will look different.